src/bci_build/package/base/README.md.j2

```diff
@@ -50,11 +50,9 @@ $ podman run -ti --rm {{ image.pretty_reference }}
 ### The SLE_BCI repository
 
 The container image comes with the free `SLE_BCI` repository. The repository
-contains a subset of all packages from SUSE Linux Enterprise. The packages are
-available free of charge, and they can be redistributed freely. However, they
-are provided without support. The repository also contains the latest version of
-packages only.
-
+contains a subset of all packages from SUSE Linux Enterprise, but only the
```

Comment:
```suggestion
provides a subset of the latest version of all packages from SUSE Linux Enterprise.
```

Reply:
together with removing the rest of the conditional sentence, which makes it imho a bit easier to read.

Reply:
this causes a bit of confusion, I tried to rephrase it.